### DISCUSSION TITLE
2007/10/10/%E6%9B%B4%E6%96%B0%E7%A4%BA%E4%BE%8B%E7%A8%8B%E5%BA%8F%E7%9A%84%E5%AD%98%E6%94%BE%E5%9C%B0%E5%9D%80

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#181] [Announcements]
# 2007/10/10/%E6%9B%B4%E6%96%B0%E7%A4%BA%E4%BE%8B%E7%A8%8B%E5%BA%8F%E7%9A%84%E5%AD%98%E6%94%BE%E5%9C%B0%E5%9D%80

前一段时间有人跟我说，我在网页上链接的范例程序都不能下载了。

https://blog.qizhen.xyz/2007/10/10/%E6%9B%B4%E6%96%B0%E7%A4%BA%E4%BE%8B%E7%A8%8B%E5%BA%8F%E7%9A%84%E5%AD%98%E6%94%BE%E5%9C%B0%E5%9D%80

**Comment by ruanqizhen** [comment]
之前博客下的留言：

shanghai_kiss
2007年10月12日 22:56 编辑
深有同感。

回复
shanghai_kiss
2007年10月12日 22:55 编辑
这就是中国。

回复
shanghai_kiss
2007年10月12日 22:55 编辑
这就是中国。
